platform_versioned_docs/version-23.3.0/monitoring/overview.mdx

```diff
@@ -1,38 +1,38 @@
+The **Runs** tab contains all previous runs in the workspace. Each new or resumed run is given a random name such as _grave_williams_. Each row corresponds to a specific run. As a run executes, it can transition through the following states:
 
 - `submitted`: Pending execution
 - `running`: Running
 - `succeeded`: Completed successfully
 - `failed`: Successfully executed, where at least one task failed with a `terminate` [error strategy](https://www.nextflow.io/docs/latest/process.html#errorstrategy)
-- `cancelled`: Stopped forceably during execution
+- `cancelled`: Stopped manually during execution
 - `unknown`: Indeterminate status
 
-Select the name of a job from the list to display that run's [execution details](./execution.mdx).
+Select the name of a run from the list to display that run's [execution details](./run-details.mdx).
 
-### Save run as pipeline
+## Save run as pipeline
 
 _Available from version 23.1_
 
-From the **Runs** list, any run can be saved as a new pipeline for future use, regardless of run status. Select the item menu to the right of any run in the list, then select **Save as pipeline**. In the dialog box shown, you can edit the pipeline name, add labels, and **Save**.
+From the **Runs** list, any run can be saved as a new pipeline for future use, regardless of run status. Select the item menu next to any run in the list, then select **Save as pipeline**. In the dialog box shown, you can edit the pipeline name, add labels, and **Save**.
 
-To review and edit any run details prior to saving the pipeline, select **Review and edit** near the top of the dialog window. When you've saved the pipeline, it's listed on the **Launchpad** and can be run from the same workspace that it was created in.
+You can **Review and edit** any run details prior to saving the pipeline. After you've saved the pipeline, it is listed on the **Launchpad** and can be run from the same workspace where it was created.
 
-### All runs view
+## All runs view
 
-_Available from version 22.4.0_
+The **All runs** page, accessed from the user menu, provides a comprehensive overview of the runs accessible to a user across the entire Seqera instance. This facilitates overall status monitoring and early detection of execution issues from a single view, split across organizations and workspaces.
 
-The **All runs** page, accessed from the user menu, provides a comprehensive overview of the runs accessible to a user across the entire Seqera instance. This facilitates overall status monitoring and early detection of execution issues from a single view split across organizations and workspaces.
+The All runs view defaults to all organizations and workspaces you can access. Select the dropdown next to **View** to filter by specific organizations and workspaces, or to view runs from your personal workspace only.
```

Comment:
If _All runs_ is a specific UI area, section, or element, in this case do we not want to use bold for it?